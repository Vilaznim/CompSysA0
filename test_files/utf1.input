UTF8 æøå: æ ø å
