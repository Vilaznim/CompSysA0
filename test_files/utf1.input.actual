test_files/utf1.input: UTF-8 Unicode text
