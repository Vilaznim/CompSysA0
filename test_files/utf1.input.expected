test_files/utf1.input: Unicode text, UTF-8 text
